doc/tutorials/building_acrn_in_docker.rst

```diff
@@ -0,0 +1,157 @@
+.. _building-acrn-in-docker:
+
+Building ACRN in Docker
+#######################
+
+In this tutorial, we will introduce how to build ACRN in Clear Linux Docker image.
+
+Install Docker
+**************
+
+#. Install Docker according to `Docker installation instructions <https://docs.docker.com/install/>`_.
+#. If you are behind HTTP or HTTPS proxy server, follow these
+   `HTTP/HTTPS proxy instructions <https://docs.docker.com/config/daemon/systemd/#httphttps-proxy>`_
+   to set up HTTP/HTTPS proxy to pull Docker image and
+   `Configure Docker to use a proxy server <https://docs.docker.com/network/proxy/>`_
+   to set up HTTP/HTTPS proxy for the Docker container.
+
+Get Docker Image
+****************
+
+This tutorial present two ways to get Clear Linux Docker image to build ACRN.
+
+Build Docker Image from Dockerfile
+==================================
+
+#. Download `Dockerfile <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/Dockerfile>`_
+   to your development machine
+#. Build Docker image
+
+   .. code-block:: none
+
+      $ docker build -t acrn/clearlinux:latest -f <path/to/Dockerfile> .
+
+   if you are behind HTTP or HTTPS proxy server, use this command instead:
+
+   .. code-block:: none
+
+      $ docker build --build-arg HTTP_PROXY=http://<proxy_host>:<proxy_port> \
+      --build-arg HTTPS_PROXY=https://<proxy_host>:<proxy_port> \
+      -t acrn/clearlinux:latest -f <path/to/Dockerfile> .
+
+
+Get Docker Image from Docker Hub
+================================
+
+You can instead, pull a pre-built Docker image from Docker Hub
+to your development machine, using this command:
+
+.. code-block:: none
+
+   $ docker pull acrn/clearlinux:latest
+
+Build ACRN from Source in Docker
```

Comment:
I think we need to change the workflow in this section. The way you do it is run the Clear Linux Docker image interactively and do everything in there (git clone, etc.). What we should do instead is let the developer do all these git operations (cloning, branching, committing, etc.) in his dev environment and use the Docker image **only** to build. So the Docker command to run becomes something like: `sudo docker run -v $PWD:/workspace acrn/clearlinux:latest bash -c "make clean && make"`